test/src/BuilderTest/ManualDocTest.js

```diff
@@ -11,135 +14,44 @@ describe('Manual:', ()=>{
   /** @test {ManualDocBuilder#_buildManualNav} */
   it('has manual navigation', ()=>{
     const doc = readDoc('manual/index.html');
-    find(doc, '[data-ice="nav"]', (doc)=>{
-      assert.includes(doc, '[data-ice="manual"]:nth-of-type(1)', 'Overview');
-      assert.includes(doc, '[data-ice="manual"]:nth-of-type(2)', 'Installation');
-      assert.includes(doc, '[data-ice="manual"]:nth-of-type(3)', 'Tutorial');
-      assert.includes(doc, '[data-ice="manual"]:nth-of-type(4)', 'Usage');
-      assert.includes(doc, '[data-ice="manual"]:nth-of-type(5)', 'Configuration');
-      assert.includes(doc, '[data-ice="manual"]:nth-of-type(6)', 'Example');
-      assert.includes(doc, '[data-ice="manual"]:nth-of-type(7)', 'Reference');
-      assert.includes(doc, '[data-ice="manual"]:nth-of-type(8)', 'FAQ');
-      assert.includes(doc, '[data-ice="manual"]:nth-of-type(9)', 'Changelog');
-
-      assert.includes(doc, '[data-ice="manual"]:nth-of-type(1) .manual-toc-title a', 'manual/overview.html', 'href');
-      assert.includes(doc, '[data-ice="manual"]:nth-of-type(2) .manual-toc-title a', 'manual/installation.html', 'href');
-      assert.includes(doc, '[data-ice="manual"]:nth-of-type(3) .manual-toc-title a', 'manual/tutorial.html', 'href');
-      assert.includes(doc, '[data-ice="manual"]:nth-of-type(4) .manual-toc-title a', 'manual/usage.html', 'href');
-      assert.includes(doc, '[data-ice="manual"]:nth-of-type(5) .manual-toc-title a', 'manual/configuration.html', 'href');
-      assert.includes(doc, '[data-ice="manual"]:nth-of-type(6) .manual-toc-title a', 'manual/example.html', 'href');
-      assert.includes(doc, '[data-ice="manual"]:nth-of-type(7) .manual-toc-title a', 'identifiers.html', 'href');
-      assert.includes(doc, '[data-ice="manual"]:nth-of-type(8) .manual-toc-title a', 'manual/faq.html', 'href');
-      assert.includes(doc, '[data-ice="manual"]:nth-of-type(9) .manual-toc-title a', 'manual/changelog.html', 'href');
-    });
+
+    var i = 1;
+    for (var file in m) {
+      if (file != 'asset') {
+
+        let label = file.charAt(0).toUpperCase() + file.slice(1);
+        let f = file.split('.')[0]+'.html';
```

Comment:
Suggest changing `let label` and `let f` to `const label` and `const f` because they are never modified.